playground/nextjs/src/components/Header.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import React from 'react';
+
+const LINKS = [
+  { href: '/', label: 'Default Inbox' },
+  { href: '/render-bell', label: 'Render Bell' },
+  { href: '/render-notification', label: 'Render Notification' },
+  { href: '/preferences', label: 'Preferences' },
+  { href: '/notifications', label: 'Notifications' },
+];
+
+const NavLink = ({ href, label }: { href: string; label: string }) => {
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** Extract type and use as typing for `LINKS` array‏